src/content/configuration/output.md

```diff
@@ -482,6 +520,31 @@ And finally the output is:
 });
 ```
 
+Note that omitting `library` will result in the assignment of all properties returned by the entry point be assigned directly to the root object, as documented under the [object assignment section](#exposing-the-library-via-object-assignment).  Example:
```

Comment:
This is pretty minor but note that we typically only use one space at the start of a new sentence. So `... assignment).  Example:` would be come `... assignment). Example:`.